.github/workflows/ci.yml

```diff
@@ -2,77 +2,86 @@ name: RSpec CI
+      - "main"
+      - "*-maintenance"
+      - "*-dev"
   pull_request:
     branches:
-      - '*'
+      - "*"
 permissions:
   contents: read
 
 jobs:
   rubocop:
     name: Rubocop
-    runs-on: 'ubuntu-20.04'
+    runs-on: "ubuntu-20.04"
     steps:
       - uses: actions/checkout@v3
       - uses: ruby/setup-ruby@v1
         with:
-          ruby-version: '3.0'
+          ruby-version: "3.0"
       - run: script/update_rubygems_and_install_bundler
       - run: bundle install --standalone
       - run: bundle binstubs --all
       - run: script/run_rubocop
 
   test:
-    name: 'Ruby: ${{ matrix.ruby }}, Rails: ${{ matrix.env.RAILS_VERSION }}'
+    name: "Ruby: ${{ matrix.ruby }}, Rails: ${{ matrix.env.RAILS_VERSION }}"
     runs-on: ubuntu-20.04
     strategy:
       fail-fast: false
       matrix:
-       include:
-         # Edge Rails (7.1) builds >= 2.7
-         - ruby: 3.1
-           allow_failure: true
-           env:
-             RAILS_VERSION: 'main'
-         - ruby: '3.0'
-           allow_failure: true
-           env:
-             RAILS_VERSION: 'main'
-         - ruby: 2.7
-           allow_failure: true
-           env:
-             RAILS_VERSION: 'main'
+        include:
+          # Edge Rails (7.1) builds >= 2.7
+          - ruby: "3.2"
+            allow_failure: true
+            env:
+              RAILS_VERSION: "main"
+          - ruby: "3.1"
+            allow_failure: true
+            env:
+              RAILS_VERSION: "main"
+          - ruby: "3.0"
+            allow_failure: true
+            env:
+              RAILS_VERSION: "main"
+          - ruby: "2.7"
+            allow_failure: true
+            env:
+              RAILS_VERSION: "main"
 
-         # Rails 7.0 builds >= 2.7
-         - ruby: 3.1
-           env:
-             RAILS_VERSION: '~> 7.0.0'
-         - ruby: '3.0'
-           env:
-             RAILS_VERSION: '~> 7.0.0'
-         - ruby: 2.7
-           env:
-             RAILS_VERSION: '~> 7.0.0'
+          # Rails 7.0 builds >= 2.7
+          - ruby: "3.2"
+            env:
+              RAILS_VERSION: "~> 7.0.0"
+          - ruby: "3.1"
+            env:
+              RAILS_VERSION: "~> 7.0.0"
+          - ruby: "3.0"
+            env:
+              RAILS_VERSION: "~> 7.0.0"
+          - ruby: "2.7"
+            env:
+              RAILS_VERSION: "~> 7.0.0"
 
-         # Rails 6.1 builds >= 2.5
-         - ruby: 3.1
-           env:
-             RAILS_VERSION: '~> 6.1.0'
-         - ruby: '3.0'
-           env:
-             RAILS_VERSION: '~> 6.1.0'
-         - ruby: 2.7
-           env:
-             RAILS_VERSION: '~> 6.1.0'
-         - ruby: 2.6
-           env:
-             RAILS_VERSION: '~> 6.1.0'
-         - ruby: 2.5
-           env:
-             RAILS_VERSION: '~> 6.1.0'
+          # Rails 6.1 builds >= 2.5
+          # Not applicable for ruby 3.2 as it is not supported.
+          # See: https://github.com/rails/rails/issues/46883#issuecomment-1371325906
```

Comment:
[memo]

>Rails 6.1 only receives security bug fixes. At the moment it doesn't work with Ruby 3.2. 
https://github.com/rails/rails/issues/46883#issuecomment-1371325906

Reply:
You don't need to mention this, we only support supported Ruby versions for currently maintained Rails